rmgpy/data/thermo.py

```diff
@@ -1720,14 +1720,23 @@ def getBicyclicCorrectionThermoDataFromHeuristic(self, bicyclic):
         # apply formula: 
         # bicyclic correction ~= saturated bicyclic correction - 
         # saturated single ring corrections + single ring corrections
+
+        estimated_bicyclic_thermodata = ThermoData(
+            Tdata = ([300,400,500,600,800,1000,1500],"K"),
+            Cpdata = ([0.0,0.0,0.0,0.0,0.0,0.0,0.0],"J/(mol*K)"),
+            H298 = (0.0,"kJ/mol"),
+            S298 = (0.0,"J/(mol*K)")
+        )
         
         saturated_bicyclic_thermoData = self.__addRingCorrectionThermoDataFromTree(None, 
                 self.groups['polycyclic'], saturated_bicyclic_submol, saturated_bicyclic_submol.atoms)[0]
+        
+        estimated_bicyclic_thermodata = addThermoData(estimated_bicyclic_thermodata,
```

Comment:
You fixed this once in e2cefaefba671d890b73414252f515c60b4fb928, then you unfixed it in 73c3e28e3fed40cacbf9136724e7997aec5cb502, then you fix it again here. I think you could probably squash the first two commits, and only fix it once here.